data_structures/linked_list/doubly_linked_list.py

```diff
@@ -107,6 +125,8 @@ def delete_value(self, value):
                 self.tail = self.tail.previous
 
             self.remove_node_pointers(node)
+        else:
```

Comment:
The function signature does not say that this function returns a string.  My sense is that we should raise an exception.